emmaa/resources/default_db_config.ini

```diff
@@ -0,0 +1,32 @@
+# Here, you may enter addresses to INDRA database instances, headed by section
```

Comment:
We can rename INDRA here and below to EMMAA

Reply:
Haha, good catch! Done.